sections/main-product.liquid

```diff
@@ -2074,20 +2074,46 @@
     },
     {
       "type": "select",
-      "id": "media_position",
```

Comment:
Media position being a bit of more general setting for that column of the section, wouldn't we want to keep it at the top ? 

I don't have strong feelings just sharing some thoughts as I go through the review 😅

Reply:
The short answer is I don't disagree with you and we've already made some adjustments based on feedback I had on the natural order these, but to be consistent across all sections we want to keep media related settings together and generally above other categories of settings, like layout (Image with text is a good example with similar settings).

"Media position" despite having the word "media" in it, is more of a general layout setting, in that the language could just as easily be inversed to read "Product info position". It doesn't explicitly effect the display of the image itself.

Reply:
True 👍 Sounds good